models/issue.go

```diff
@@ -97,6 +97,9 @@ func (issue *Issue) loadTotalTimes(e Engine) (err error) {
 
 // IsOverdue checks if the issue is overdue
 func (issue *Issue) IsOverdue() bool {
+	if issue.IsClosed {
```

Comment:
If you close an issue before deadline it will be marked overdue

shouldn't we just return false if IsClosed?

Reply:
I think I mentioned that in the issue:
Basically there are two possible options for closed issues/PRs -
a) we simply hide it/ show it as not overdue
b) we differentiate between whether it has been closed on time or not.
Both are in my opinion valid. The currently suggested approach is b)

What I am asking myself is:
Is that function also getting called when rendering the PR conversation page where the due date gets displayed?
Because if that is the case then I would say we should stick to the currently suggested behavior, because that is the one place where it certainly should always get displayed correctly

Reply:
@6543 how are you getting if you close an issue before its deadline it will be marked overdue for this test? 

`issue.ClosedUnix >= issue.DeadlineUnix
`

Reply:
There is a question of what happens if you reopen and then reclose an issue. That would make the issue overdue

Reply:
ah sure ... missed that one